src/mediaelement.js

```diff
@@ -89,7 +124,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     seekTo: function (start) {
-        if (start != null) {
+        if (start) {
```

Comment:
Why did you change this? If start is `0` we want to go to the beginning, not to the current position.